integration_test/list_test.go

```diff
@@ -15,11 +15,16 @@
 package integrationtest
 
 import (
+	"encoding/json"
+	"path/filepath"
+	"regexp"
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
+	"gopkg.in/yaml.v2"
 )
 
+// FIXME: still necessary?
```

Comment:
yeah i think we'd have a very preliminary "list" test and this can be it.

Reply:
Shouldn't line 29 be removed then since its only ever run as part of a larger battery?